### ISSUE TITLE
don't mention estimate if it's not present

### ISSUE THREAD

**Issue by yegor256** [yegor256/0pdd#70]
In `github_tickets.rb` we are saying 

```
Estimate: #{puzzle.xpath('estimate')[0].text} minutes, \
role: #{puzzle.xpath('role')[0].text}.\
```

Let's skip the estimate if it's equal to 0 and skip the role if it's equal to `IMP`.

**Comment by yegor256** [comment]
@rultor release, tag is `0.18.1`

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/0pdd#70` was not in scope

**Comment by yegor256** [comment]
@0crat assign @tedtoer

**Comment by 0crat** [comment]
> @0crat assign @tedtoer  ([here](https://github.com/yegor256/0pdd/issues/70#issuecomment-336805335))

@yegor256 Job `gh:yegor256/0pdd#70` assigned to @tedtoer, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)).

**Comment by 0crat** [comment]
Bug was reported: +15 points just awarded to @yegor256, total is [+4070](http://www.0crat.com/u/yegor256).

**Comment by 0crat** [comment]
@tedtoer this job was assigned to you 8 days ago.
It will be taken away from you after 10 days from start (this is our policy).

**Comment by 0crat** [comment]
@tedtoer resigned from `gh:yegor256/0pdd#70`, please stop working.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/0pdd#70` is not assigned to anyone.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/0pdd#70` is not assigned to anyone.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/0pdd#70` is not assigned to anyone.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/0pdd#70` is not assigned to anyone.

**Comment by 0crat** [comment]
@tedtoer resigned from `gh:yegor256/0pdd#70`, please stop working.

**Comment by 0crat** [comment]
Job `gh:yegor256/0pdd#70` assigned to @tedtoer. The budget is [fixed](http://datum.zerocracy.com/pages/policy.html#4) and it is 30 minutes. Please, read the [Policy](http://datum.zerocracy.com/pages/policy.html) and go ahead.

**Comment by 0crat** [comment]
@tedtoer resigned from `gh:yegor256/0pdd#70`, please stop working.

**Comment by SilasReinagel** [comment]
@0crat wait on PR to merge

**Comment by 0crat** [comment]
> @0crat wait on PR to merge ([here](https://github.com/yegor256/0pdd/issues/70#issuecomment-381444969))

@SilasReinagel The impediment for [#70](https://github.com/yegor256/0pdd/issues/70) was registered successfully by @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
@rultor[/z](https://www.0crat.com/u/rultor) the issue is closed not by @yegor256[/z](https://www.0crat.com/u/yegor256) (its creator); I won't close the order; please, re-open it and ask @yegor256[/z](https://www.0crat.com/u/yegor256) to close it

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@94404d3d-143d-4878-8b2e-156985fa2aa4, thread: AsyncFlush-6
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/close_job.groovy
sun.reflect.NativeMethodAccessorImpl[-2] java.lang.reflect.InvocationTargetException: null
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by SilasReinagel** [comment]
@yegor256 Apparently this PR still uses the GitHub autoclose feature. I will avoid it in the future.

In the meantime, can you please close and re-open this?

**Comment by yegor256** [comment]
@rultor release, tag is `0.30.1`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@c7eb7d17-4a8c-4bbc-8e74-40a0095a9724, thread: BkParallel-9
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
The job [#70](https://github.com/yegor256/0pdd/issues/70) is now out of scope

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order
